src/components/forms/fields/InputFieldWithTags/InputFieldWithTags.tsx

```diff
@@ -110,23 +110,20 @@ export const InputFieldWithTags = (props: InputFieldWithTagsProps) => {
           {label}
         </label>
       }
-      <div className={cl['bk-input-field-with-tags__container']}>
+      <Input
+        {...inputProps}
+        id={controlId}
+        form={formContext.formId}
+        className={cx(cl['bk-input-field-with-tags__control'], inputProps.className)}
+        onChange={onChange}
+        onKeyDown={onKeyDown}
+        value={value}
+      />
+      <div className={cl['bk-input-field-with-tags__tags']}>
         {tags && (
           // biome-ignore lint/suspicious/noArrayIndexKey: no other unique identifier available
           tags.map((tag, idx) => <Tag key={idx} content={tag} onRemove={() => onRemoveTag(idx)}/>)
```

Comment:
This is more of an issue in the `Tag` component itself, but the close button is not keyboard navigable at the moment. Need to wrap the close icon in a `<Button unstyled>` element and move the `onClick` to an `onPress` on the button.